contrib/cirrus/lib.sh

```diff
@@ -345,6 +345,21 @@ install_test_configs(){
 remove_packaged_podman_files(){
     echo "Removing packaged podman files to prevent conflicts with source build and testing."
     req_env_var OS_RELEASE_ID
+
+    # If any binaries are resident they could cause unexpected pollution
+    for unit in io.podman.service io.podman.socket
```

Comment:
It looks pretty likely that the service names are the same on Ubuntu, but I can't confirm because I have no access to said OS. Can we guarantee that this is the case?

Reply:
`hack/get_ci_vm.sh` should work for you no?

Yes I will confirm...

Reply:
...confirmed.  It's same-same.